docs/README.md

```diff
@@ -172,6 +174,23 @@ CORECLR_PROFILER_PATH_64
 OTEL_DOTNET_AUTO_INTEGRATIONS_FILE
 ```
 
+You can also use the [instrument.sh](../instrument.sh) script which uses following
+environment variables as parameters:
+
+| Parameter          | Description                                                            | Required | Default value        |
+|--------------------|------------------------------------------------------------------------|----------|----------------------|
+| `DISTRIBUTION`     | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows`       | Yes      |                      |
+| `ENABLE_PROFILING` | Whether to set the .NET CLR Profiler, possible values: `true`, `false` | No       | `true`               |
```

Comment:
I'm not sure if it is going to get too verbose to add it here, but, the important fact in this case is that bytecode instrumentation requires profiling.

Reply:
We can improve the text here later.

Reply:
It is described ~20 lines above. I know that it is not ideal 😉

Reply:
Let's work to improve it later.